### PR TITLE
Release 252.0.0

### PR DESCRIPTION
**Pull request by bergeron**
Minor release of assets controllers to include https://github.com/MetaMask/core/pull/4949

### CODE REVIEW

File: packages/assets-controllers/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [44.1.0]
+
+### Changed
+
+- An argument `networkClientId` is added to `TokensController.ignoreTokens`, allowing tokens to be ignored on specific chains.
+
```
Comment (mcmire) [inline packages/assets-controllers/CHANGELOG.md:15]: Looks like this is missing a PR link. In addition there is an extra line here:

``` suggestion
- An argument `networkClientId` is added to `TokensController.ignoreTokens`, allowing tokens to be ignored on specific chains. ([#4949](https://github.com/MetaMask/core/pull/4949))

File: packages/assets-controllers/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Bump `@metamask/{keyring-api,eth-snap-keyring,snaps-*}` ([#4948](https://github.com/MetaMask/core/pull/4948))
```
Comment (mcmire) [inline packages/assets-controllers/CHANGELOG.md:16]: It looks like this was a bump to dev dependencies:

``` suggestion
```

**Review by mcmire — COMMENTED** [review]
Two things but otherwise looks good.

**Review by gambinish — DISMISSED** [review]
Preemptively approving. Lmk if you need another stamp after @mcmire changes

**Pull request merged**